R/utils.R

```diff
@@ -18,6 +18,41 @@ source_GitHubData <- function(url, sep = ",", header = TRUE) {
   read.table(handle, sep = sep, header = header)
 }
 
+##' Calculate RMSE and percent RMSE with PLSR model results
+##' 
+##' @param plsr_dataset input plsr dataset
+##' @param inVar the trait variable used in the calculation of RMSE
+##' @param residuals predicted minus observed residual vector from either 
+##' a cross-validation CV or independent validation
+##' @param range calculate over the full data range or the 95% of data range. 
+##' options full or 95perc
+##' @return output a list containing the rmse and perc_rmse. 
+##' output <- list(rmse = rmse, perc_rmse = perc_rmse)
+##' 
+##' @author Shawn P. Serbin
+##' 
+##' @export
+percent_rmse <- function(plsr_dataset = NULL, inVar = NULL, 
```

Comment:
@JulienLamour created this function for calculating RMSEP and %RMSEP and it matches that in Feilhauer and Asner now.  Updated the scripts as well to use this